app/templates/components/hyper-table/cell-renderers/money.hbs

```diff
@@ -1,5 +1,11 @@
-{{#if (and emptyAmount emptyCurrency)}}
-  {{format-money amount currency}}
-{{else}}
-  —
-{{/if}}
+<div class="editing-input" style="">
+  {{#if isEditing}}
+    {{input focus-out=(action "toggleEditing") type="text" value=amount class="editing-input__field form-control"}}
+    <i class="fa fa-save margin-left-xx-sm" onclick={{action "toggleEditing"}}/>
+  {{else}}
+    <span data-toggle="tooltip" data-title={{format-money amount currency}}>
```

Comment:
Whats happen with empty value ?

Reply:
Same, it will just turn on the empty case